bee-node/src/node.rs

```diff
@@ -46,27 +51,25 @@ pub enum Error {
     ShutdownError(#[from] bee_common::shutdown::Error),
 }
 
-pub struct NodeBuilder {
-    config: NodeConfig,
+pub struct NodeBuilder<B: Backend> {
+    config: NodeConfig<B>,
+    phantom: PhantomData<B>,
```

Comment:
Is this needed since the config is generic over B ?

Reply:
Ah, good point. Another refactoring artifact!